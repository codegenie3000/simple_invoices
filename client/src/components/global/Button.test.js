import React from 'react';
import { shallow, render } from 'enzyme';

import ButtonPrimary from './Button';

describe('Button Primary', () => {
    it('should render correctly', () => {
        const component = render(<ButtonPrimary buttonText={'foo'}/>);
        expect(component).toMatchSnapshot();
    })
});
