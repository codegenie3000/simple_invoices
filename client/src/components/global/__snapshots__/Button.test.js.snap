// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Primary should render correctly 1`] = `
<button
  class="buttonPrimary"
  style="margin:10px auto auto 0"
  type="submit"
>
  foo
</button>
`;
